pom.xml

```diff
@@ -1681,6 +1681,14 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
```

Comment:
please move this to the jooby-akka/pom.xml

Reply:
done.